hydrus/data/exceptions.py

```diff
@@ -174,3 +174,11 @@ def get_HTTP(self) -> HydraError:
         """Return the HTTP response for the Exception."""
         description = "Offset {} is out of range.".format(self.offset)
         return HydraError(code=400, title="Page not found", desc=description)
+
+
+class ClientTooOutdated(Exception):
```

Comment:
Again same here. Sync mechanism is not a part of Hydra, we don't use HydraError for this. This should be served using the socket, not the app.